internal/controller/kubeobjects/velero/requests.go

```diff
@@ -28,6 +28,8 @@ const (
 	path         = "velero/"
 	protectsPath = path + "backups/"
 	recoversPath = path + "restores/"
+
+	drCConfigCreatedByLabel = "ramendr.openshift.io/resource-created-by-ramen"
```

Comment:
This label is duplicated, there should be exactly one variable shared by all code using it and maybe the test expecting this value.

This should also be a constant since it should never change.